vendor/fmt/.clang-format

```diff
@@ -0,0 +1,8 @@
+# Run manually to reformat a file:
```

Comment:
Just curious re the decision to copy the source code here. Is that the simplest way to do it?

Reply:
Sadly the only way to do it -- swiftpm can check out other repositories if they also have a swiftpm build.  There is _some_ support for `brew install` but it is not encapsulated then.

All of the dependencies have to be copied in this way.